lib/SILOptimizer/Mandatory/TFUtilities.h

```diff
@@ -115,26 +115,39 @@ struct SILTensorOpInfo {
   /// input that is an array of tensors.  An integer attribute may be either
   /// a Tensor value or an integer-encoded DType, etc.
   enum class OperandClass {
-    /// This marks the following sorts of things:
+    /// Indicates one of the following:
```

Comment:
This list is out-dated? It might be nice if we have a shorter and high level summary here, and rely on per enum-comments below to describe what they are.

Reply:
Sorry, I missed this comment!

It's likely outdated. @marcrasi could you check if the description of `Input` can be updated when you implement input unpacking?

Reply:
yep, will do